doc/administrator/web/ubuntu/index.md

```diff
@@ -6,25 +6,65 @@ description: "Ubuntu-specific install guide for setting up IRIDA."
 
 Ubuntu Installation Guide
 =========================
-Starting with a fresh, completely up-to-date system (Ubuntu Server 18.04).
+Starting with a fresh, completely up-to-date system (Ubuntu 24.04).
 
 Installing Software with `apt`
 ------------------------------
-You can install the latest OpenJDK JDK on Ubuntu. The `apt` repositories also contain an up-to-date version of Tomcat 8, so no complex configuration needs to take place for Ubuntu.
 
-    #### Install OpenJDK Java 11 
-    apt-get install openjdk-11-jdk
+    #### Install OpenJDK Java 17
+    sudo apt-get install openjdk-17-jdk
 
-    #### Install tomcat8 and mariadb-server
-    apt-get install --yes tomcat8 libtcnative-1 mariadb-server
+    #### Install mariadb-server
+    sudo apt-get install --yes mariadb-server
+
+    #### Download and Install tomcat9
+    sudo mkdir /opt/tomcat
+
+    For security reasons, it's best not to run Tomcat as the root user so create a dedicated user and group
+    sudo useradd -m -U -d /opt/tomcat -s /bin/false tomcat
+
+    Download [Tomcat 9](https://tomcat.apache.org/download-90.cgi)

```

Comment:
**question:** I'm assuming I am downloading from the **core** menu?

Reply:
Correct!